add get information about a column of a modern page command, solving #412

RARomano opened the pull request.

waldekmastykarz commented on docs/manual/docs/cmd/spo/page/page-column-get.md:
```diff
@@ -0,0 +1,39 @@
+# spo page column list
```
This should be `spo page column get`

waldekmastykarz commented:
@RARomano when getting information about a column, I think it would be valuable to get the list of controls in that particular column, so that you can better manipulate its contents. What do you think?

RARomano commented:
I agree with you, I’ll adjust it and include the fixes you pointed out as well. 
I used an existing template and I forgot to change the name in some spots, sorry for that.

RARomano commented:
About your previous comment, should I add the informations about the controls to all commands that either get or list columns ?

As we're using a single method `getColumnsInformation` to return its information, it would be an easy change and it would keep the responses consistent

waldekmastykarz commented:
When I think about how you'd use the commands, I think it would make the most sense to show the info only for `column get`. For `column list` you want to know what columns are in the particular section and what factor they have. Optionally, we could output the number of controls so that you can more easily find the column you're after.

RARomano commented:
@waldekmastykarz I've fixed the issues as you requested and I have split the other changes in different commits to make it easier to read:
- added controls count (`controls` property to the `page column list command`)
- refactored the page control get command to make the code more reusable
- added the controls information to the `page column get` command

waldekmastykarz commented:
Cool! Thanks and appreciate your changes. I'll review the PR shortly

waldekmastykarz commented:
Merged manually. Thank you!
